internal/test/wmcb/wmcb_test.go

```diff
@@ -713,4 +720,37 @@ func (f *pkgInfo) getSHAFileContent(url string)(string, error){
 		checksumFileContent = scanner.Text()
 	}
 	return  checksumFileContent, nil
-}
\ No newline at end of file
+}
+
+// initializeKubeNodePkgInfo populates the fileds of KubeNode pkg
+func (f *KubeNode) initializePkgInfo() error {
+	var kubeNodePkg KubeNode
+	if k8sVersion := framework.K8sVersion; k8sVersion == "" {
+		return fmt.Errorf("kubernetes version is empty")
+	}
+	kubeNodePkg.url = kubeNodeBaseURL + framework.K8sVersion + "/kubernetes-node-windows-amd64.tar.gz"
+
+	kubeNodeSHA, err := kubeNodePkg.getSHAValue(kubeNodePkg.url)
+	if err != nil {
+		return fmt.Errorf("unable to get SHA for Kubernetes node binary for Windows: %v", err)
+	}
+	kubeNodePkg.sha = kubeNodeSHA
+
+	kubeNodePkg.shaType = "sha512"
+	f.pkgInfo = kubeNodePkg.pkgInfo
+	return nil
+}
+
+// getKubeNodeSHA returns the SHA512 value of the kube node package given the package url.
+func (f *KubeNode) getSHAValue(kubeNodeURL string) (string, error) {
+	checksumFileContent, err := f.getSHAFileContent(kubeNodeURL)
+	if err!= nil{
+		return "", err
+	}
+	// the checksum file content contains a single value in the "<sha>" format.
+	sha512 := strings.Split(checksumFileContent, "\n")
```

Comment:
Why are you splitting if checksum file content has only <sha>. Are you trying to remove '\n' ? Can you use [`strings.TrimSpace`](https://golang.org/pkg/strings/#TrimSpace) instead?

Reply:
I think she is trying to split the string on "\n"